src/Microsoft.AspNetCore.OData/Routing/Conventions/OperationRoutingConvention.cs

```diff
@@ -172,6 +180,15 @@ protected void ProcessOperations(ODataControllerActionContext context, IEdmEntit
             }
         }
 
+        private static IEdmOperation[] FindCandidates(ODataControllerActionContext context, string operationName)
+        {
+            // TODO: refactor here
+            // If we have multiple same function defined, we should match the best one?
+
+            StringComparison actionNameComparison = context.Options?.RouteOptions?.EnableActionNameCaseInsensitive == true ? StringComparison.OrdinalIgnoreCase : StringComparison.Ordinal;
+            return context.Model.SchemaElements.OfType<IEdmOperation>().Where(f => f.IsBound && f.Name.Equals(operationName, actionNameComparison)).ToArray();
```

Comment:
Long lines like this are really hard to read.

I'd suggest reformatting like:
```suggestion
            return context.Model.SchemaElements
                .OfType<IEdmOperation>()
                .Where(f => f.IsBound && f.Name.Equals(operationName, actionNameComparison))
                .ToArray();
```

Reply:
#Resolved.